### PR TITLE
Add xread block and count support

### PR DESCRIPTION
**Pull request by jasonatball**

### CODE REVIEW

File: spec/commands/xread_spec.rb [hunk 1]
```diff
@@ -47,4 +47,20 @@
     expect(@redises.xread([@key, @key1], %w[1234567891234-2 1234567891234-1]))
       .to eq({ @key1 => [['1234567891234-2', { 'key1' => 'value2' }]] })
   end
+
+  it 'supports the block parameter' do
```
Comment (jasonatball) [inline spec/commands/xread_spec.rb:51]: @sds I accepted the block parameter but don't do anything with it. Do you implement blocking anywhere else in the mock_redis? I had considered just doing a ```sleep block/1000.0``` but didn't see sleep anywhere else in the code base.

Comment (sds) [reply spec/commands/xread_spec.rb:51]: We will rely on users of this library to report any issues with what you've implemented, and provide a PR fixing the logic to add the support if it is needed. Thank you!

**Comment by jasonatball** [comment]
@sds could you create a new release which incorporates this change?

**Comment by sds** [comment]
[0.26.0](https://rubygems.org/gems/mock_redis/versions/0.26.0) has been released.

**Pull request merged**